### ISSUE TITLE
Multilingual tags for blog posts

### ISSUE THREAD

**Issue by skirsdeda** [nephila/djangocms-blog#109]
I see that django-taggit is the defacto standard for tagging in django and there's nothing else maintained in this category. But it kind-of doesn't fit the whole picture where everything is supposed to be translatable. I mean if categories can be translated, tags should be translatable as well.
What could be done with this or is it not worth the effort? :)

**Comment by yakky** [comment]
That's still an unsolved question, actually :)
probably the best solution is to use django-modeltranslation (https://github.com/deschler/django-modeltranslation) to add translations to taggit

**Comment by zanderle** [comment]
Wouldn't it make more sense to have separate tags for each language (vs translating them)?

**Comment by yakky** [comment]
That's another option. Actually the way I use it now

**Comment by zanderle** [comment]
@yakky how did you implement that? Do you also make sure that on each translation, only tags from that languages are shown?

**Comment by yakky** [comment]
No, I mean that having different tags for different languages is the way I use, without trying to translate the tags

**Comment by benebun** [comment]
@yakky I guess you used one tag list for each language in the same model like django-modeltranslation does it with other fields? (title_en, title_de etc.)? Did you use django-modeltranslation in combination with django-taggit or did you roll your own to work with taggit?

**Comment by yakky** [comment]
@benebun I've actually never used multilingual for taggit or translated tags on the blog

**Comment by benebun** [comment]
@yakky Oh ok I think I misunderstood your comment above then.
I don't know how djangocms-blog is organised yet (separate page tree per language vs. inline translation on a page) so I should probably look into that first to understand how you implemented it.

**Comment by yakky** [comment]
@benebun djangocms blog uses parler, but it's not applicable to tags (it requires the model to be changed to handle that). In theory django_modeltranslation is the perfect solution for a case like this and it should no require any specific code into the blog

**Comment by benebun** [comment]
Yup thanks, we will try it that way.

**Comment by vnavascues** [comment]
Hi guys,

I'm trying to simulate "multilingual tags" creating different tags for different languages. Let's put an example:
1. Imagine I have a Post (with ID 1) in both languages: en, es. But I leave tags in blank.
2. Go to Django Admin > Taggit > Tags > **Add Tag**.
3. Create it:  
* Name: tag1_en
* Object id: 1
* Content type: blog article
4. Create a new tag:
* Name: tag1_es
* Object id: 1
* Content type: blog article Translation

But post only displays **tag1_en** in both languages. 
Changing the strategy and setting the tags by post edit interface... If I edit the post in english and set **tag1_en** both languages have the same tag. If I swap to the other language tab and set **tag1_es**, both languages have the same tag.

Many thanks
